### DISCUSSION TITLE
collapse operator or lindblad dissipator

### DISCUSSION THREAD

**Discussion by 4psireal2** [qutip/qutip#2282] [Q&A] [answered]
I'm doing a simulation of a dissipative chain. Based on the [documentation](https://qutip.org/docs/latest/guide/dynamics/dynamics-master.html?highlight=collapse%20operator#the-lindblad-master-equation) , `c_ops` is simply a list of `C_n`. I also came across the [lindblad_dissipator](https://qutip.org/docs/latest/apidoc/functions.html?highlight=lindblad_dissipator#qutip.superoperator.lindblad_dissipator) function. I tested out the 2 approaches and the results are different.

```
dissipation_op_1 = []
for i in range(L):
	dissipation_op.append(self.gamma * lindblad_dissipator(a=destroy_list[i]))

dissipation_op_2 = []
for i in range(L):
	dissipation_op_2.append(self.gamma * destroy_list[i])

result_1 = mesolve(H, state0, tlist=self.time, c_ops=dissipation_op_1, options=options, progress_bar=True)
result_2 = mesolve(H, state0, tlist=self.time, c_ops=dissipation_op_2, options=options, progress_bar=True)
```

It seems to me that `dissipation_op_1` is the way to define `c_ops` for wavefunction method while `dissipation_op_2` is for density matrix. Any comments / feedbacks are very much appreciated.

**Comment by nwlambert** [comment, marked as answer]
The issue is probably because method 2 requires you to take the square root of the rate gamma, see just below equation 3 in the documentation and the example code there.

The logic is that the ''collapse operator'' in that approach is actually c = sqrt(gamma) * a,   which will give the lindblad dissipator equivalent to method 1 when the construction of the dissipator is made internally in mesolve(), i.e., c \rho c^{\dagger}+... =  gamma a \rho a^{\dagger}....
